src/Compilers/Core/Portable/PEWriter/SigningUtilities.cs

```diff
@@ -10,6 +10,7 @@
 using System.Security.Cryptography;
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Emit;
+using Microsoft.CodeAnalysis.Symbols;
```

Comment:
Am I missing the reference changed that requires this namespace add?

Reply:
> Am I missing the reference changed that requires this namespace add?

I moved some types used in this file to this namespace

---
In reply to: [339148226](https://github.com/dotnet/roslyn/pull/39498#discussion_r339148226) [](ancestors = 339148226)